addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoHandlerFactory.java

```diff
@@ -94,28 +101,56 @@ protected ThingHandler createHandler(Thing thing) {
         }
     }
 
+    @Override
+    protected void removeHandler(ThingHandler thingHandler) {
+        if (thingHandler instanceof NetatmoBridgeHandler) {
+            unregisterDeviceDiscoveryService(thingHandler.getThing().getUID());
```

Comment:
Maybe only one call for `thingHandler.getThing().getUID()`? It's used twice here.

Reply:
Done